.github/workflows/prebuild.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     name: ${{ matrix.canvas_tag}}, Node.js ${{ matrix.node }}, Linux
     runs-on: ubuntu-latest
     container:
-      image: chearon/canvas-prebuilt:7
+      image: chearon/canvas-prebuilt:9
```

Comment:
I've been using the `prebuilds` branch to avoid tons of noise on `master`. Just need to remember to change that branch before the next release.

Reply:
ah, no change needed then. it's already using v9